README.md

```diff
@@ -1 +1,49 @@
-# openapi-ts-template
\ No newline at end of file
+# openapi-ts-template
+
+## Description
+
+A `Node.js` typescript template with open api implementation 
+
+## Configuration
+
+Use a `.env` at root of the repository to set values for the environment variables defined in `.env` file. 
+
+| variable         | required |   default   | description                                                                          |
+| :--------------- | :------: | :----------------------: | :----------------------------------------------------------------------------------- |
+| PORT             |    N     |   `3000`                 | The port for the API to listen on                                                    |
+| LOG_LEVEL        |    N     |   `debug`                | Logging level. Valid values are [`trace`, `debug`, `info`, `warn`, `error`, `fatal`] |
+| ENVIRONMENT_VAR  |    N     |  `example`               | An environment specific variable                                                     |
+| DB_PORT          |    N     |   `5432`                 | The port for the database                                                            |
+| DB_HOST          |    N     |   `localhost`            | The database hostname / host                                                         |
+| DB_NAME          |    N     |   `openapi-ts-template ` | The database name                                                                    |
+| DB_USERNAME      |    N     |   `postgres`             | The database username                                                                |
+| DB_PASSWORD      |    N     |   `postgres`             | The database password                                                                |
+
+## Getting started
+
+```sh
+# start dependencies
+docker compose up -d
+# install packages
+npm i
+# generate swagger and route files for Open API
+npm run build
+# start service in dev mode. In order to start in full - npm start"
```

Comment:
missing a `npm run db:migrate`?

Reply:
indeed. This is why I would like somebody else to take a look because I'm already in this mind where I see very little "when action becomes not profitable - sleep"